src/detect.h

```diff
@@ -1021,7 +1021,7 @@ typedef struct HttpReassembledBody_ {
 typedef struct SignatureNonPrefilterStore_ {
     SigIntId id;
     SignatureMask mask;
-    uint8_t alproto;
+    AppProto alproto;
```

Comment:
does this change the size of the struct? guess not if `AppProto` is uint16_t, but if it is an enum it would

Reply:
`typedef uint16_t AppProto;` ;-)